### PR TITLE
Displaying resource type when deleting item

### LINKED ISSUE #3812: Resource deletion should specify resource type in confirmation text
When deleting a silence, the confirmation message doesn't confirm it's a silence, and in the (fairly common) instance of all checks being silenced for an entity (using entity subscription) the wording looks a little disconcerting `"are you sure you would like to delete resource 'entity:your-entity'"`

## Expected Behavior

```
sensuctl silenced delete
? Subscription: entity:www.google.com
? Check:
? Are you sure you would like to delete silence resource 'entity:www.google.com:*'?
or
? Are you sure you would like to delete silence 'entity:www.google.com:*'?
```

## Current Behavior

```
sensuctl silenced delete
? Subscription: entity:www.google.com
? Check:
? Are you sure you would like to delete resource 'entity:www.google.com:*'?
```

## Possible Solution
Add the name of the resource type in the confirmation text.

### PR DESCRIPTION
**Pull request by agm650**
Signed-off-by: Luc Dandoy <luc.dandoy@gmail.com>

## What is this change?

Displaying resource type in the deletion confirmation prompt.

## Why is this change necessary?

Proposal to fix #3812.

## Does your change need a Changelog entry?

An entry was added to the changelog file

## Do you need clarification on anything?

No.

## Were there any complications while making this change?

Not really. Not sure if my fix is the most "efficient"/"beautiful" one

## Have you reviewed and updated the documentation for this change? Is new documentation required?

I don't think so as it's mainly a "cosmetic" change

## How did you verify this change?

I've been deleting various resource on a test environment to check if the new prompt was correctly displayed.

## Is this change a patch?

no

### CODE REVIEW

**Review by echlebek — CHANGES_REQUESTED** [review]
I like the change! A small nit, it should be "ConfirmDeleteResource", not "ConfirmDeleteRessource". Would you mind fixing that? Thanks!

**Comment by agm650** [comment]
I'll do that and update the request

File: cli/commands/clusterrole/delete.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ func DeleteCommand(cli *cli.SensuCli) *cobra.Command {
 			}
 			name := args[0]
 			if skipConfirm, _ := cmd.Flags().GetBool("skip-confirm"); !skipConfirm {
-				if confirmed := helpers.ConfirmDelete(name); !confirmed {
+				if confirmed := helpers.ConfirmDeleteResource(name, "clutser-role"); !confirmed {
```
Comment (echlebek) [inline cli/commands/clusterrole/delete.go:26]: Oops, one more typo here!

Comment (agm650) [reply cli/commands/clusterrole/delete.go]: solved!

**Review by echlebek — APPROVED** [review]

**Pull request merged**